zcutil/build.sh

```diff
@@ -73,9 +76,17 @@ then
     shift
 fi
 
+# If --disable-rust is the next argument, disable Rust code:
```

Comment:
In #2120 I added a `CONFIGURE_FLAGS` env var for passing flags to `./configure`; I think that's a more extensible way to proceed.